audio/wavpack/Portfile

```diff
@@ -32,7 +32,7 @@ platform darwin {
     # handrolled asm has issues on older platforms.  Disable it if we don't have access to newer compilers.
     # Note that "libc++" is really a proxy here for "configuration that supports newer compilers"; the C++
     # runtime actually isn't relevant other than being a requirement of clang-3.5+.
-    if {${configure.cxx_stdlib} eq "libc++"} {
+    if {${configure.cxx_stdlib} eq "libc++" && ${build_arch} ne "arm64"} {
```

Comment:
This change is not correct for the universal variant.

Reply:
Hey, I'm new to MacPorts. Can you elaborate on what I should be doing here? Thanks.

Reply:
v